specs/http-api/README.md

```diff
@@ -73,12 +70,13 @@ Version: Draft
 # Discoverability
 PFIs can become publicly discoverable by advertising their API endpoint as a [Service](https://www.w3.org/TR/did-core/#services) within their DID Document. In order to increase the likelihood of being discovered The `service` entry **SHOULD** include the following properties:
 
-| Property          | Value                                                                                       |
-| ----------------- | ------------------------------------------------------------------------------------------- |
-| `id`              | see [DID-CORE spec](https://www.w3.org/TR/did-core/#services)                               |
-| `type`            | `PFI`                                                                                       |
-| `serviceEndpoint` | PFI's publicly addressable API endpoint or DID which has PFIs publicly addressable endpoint |
-If the serviceEndpoint is itself a DID, this DID should resolve to a document and then its serviceEndpoints can be examined for the `PFI` type entry. The ID can be chosen at the discretion of the PFI, but the service entry should be of type `PFI`.
+| Property          | Value                                                         |
+| ----------------- | ------------------------------------------------------------- |
+| `id`              | see [DID-CORE spec](https://www.w3.org/TR/did-core/#services) |
+| `type`            | `PFI`                                                         |
+| `serviceEndpoint` | PFI's publicly addressable API endpoint                       |
```

Comment:
could we link this API endpoint to something?

Reply:
🤔 like to a spec?

Reply:
something like a common base API https://swagger.io/docs/specification/api-host-and-base-path/

Reply:
done https://github.com/TBD54566975/tbdex/pull/370/commits/766aba6830c29d096fad65b79c39064c7af6cbb4